Godeps/_workspace/src/github.com/square/go-jose/jws.go

```diff
@@ -47,9 +47,9 @@ type JsonWebSignature struct {
 // Signature represents a single signature over the JWS payload and protected header.
 type Signature struct {
 	Header    JoseHeader
+	Signature []byte
```

Comment:
This doesn't appear to be in the go-jose codebase. If we're forking go-jose, let's move it out of Godeps so that the next person that needs to update go-jose doesn't stomp on the changes.

Upstreaming this, of course, is preferred.